channeldb/payment_control.go

```diff
@@ -74,6 +76,14 @@ var (
 	// errNoAttemptInfo is returned when no attempt info is stored yet.
 	errNoAttemptInfo = errors.New("unable to find attempt info for " +
 		"inflight payment")
+
+	// errNoIndexBucket is returned when an attempt is made to lookup an
+	// index when the bucket containing indexes does not exist.
+	errNoIndexBucket = errors.New("no index bucket")
+
+	// errNoIndex is returned when an attempt to lookup a payment index is
+	// made for a sequence number that is not indexed.
+	errNoIndex = errors.New("index does not exist")
```

Comment:
ambiguous as to _which_ index/bucket is missing, e.g. payment index (also above)